packages/create-gatsby/src/questions.json

```diff
@@ -0,0 +1,55 @@
+[
+  {
+    "type": "input",
+    "name": "project",
+    "message": "What would you like to name the folder where you site will be created?",
+    "initial": "my-gatsby-site"
+  },
+  {
+    "type": "select",
+    "name": "cms",
+    "message": "Will you be using a CMS?",
+    "hint": "Use arrow keys to move, and enter to select",
+    "choices": [
+      { "message": "No (or I'll add it later)", "value": "" },
+      { "message": "WordPress", "value": "wordpress" },
+      { "message": "Contentful", "value": "contentful" },
+      { "message": "DatoCMS", "value": "datocms" },
+      { "message": "Sanity", "value": "sanity" },
+      { "message": "AgilityCMS", "value": "agility" }
+    ]
+  },
+  {
+    "type": "select",
+    "name": "styling",
+    "message": "Would you like to install a styling system?",
+    "hint": "Use arrow keys to move, and enter to select",
+    "choices": [
+      { "message": "No (or I'll add it later)", "value": "" },
+      { "message": "CSS Modules/PostCSS", "value": "postcss" },
+      { "message": "Styled Components", "value": "styled-components" },
```

Comment:
```suggestion
      { "message": "styled-components", "value": "styled-components" },
```

Total nitpick. 🙈

Reply:
If anyone is allowed to nitpick there then you can 😂